decidim-admin/app/views/layouts/decidim/admin/settings.html.erb

```diff
@@ -4,6 +4,9 @@
       <%= t ".title" %>
     </div>
     <ul>
+      <li <% if is_active_link?(decidim_admin.edit_organization_external_domain_whitelist_path) %> class="is-active" <% end %>>
```

Comment:
I would move this link to the very bottom of the admin organization settings links. I believe it should have a lower priority in the list.